Cargo.toml

```diff
@@ -44,6 +44,7 @@ time = { version = "0.3", default-features = false, optional = true, features =
 ipnetwork = { version = "0.20", default-features = false, optional = true }
 mac_address = { version = "1.1", default-features = false, optional = true }
 ordered-float = { version = "3.4", default-features = false, optional = true }
+sqlx = { git = "https://github.com/yasamoka/sqlx", branch = "pg-interval-hash", default-features = false, optional = true }
```

Comment:
There was previously no dependency on SQLx.  However, `PgInterval` is provided by SQLx. Is this fine?